### ISSUE TITLE
adding a FAQ's and   LiveChat to the website

### ISSUE THREAD

**Issue by comfy96** [unstructuredstudio/zubhub#664]
I noticed the website doesn't have 'frequently asked questions and a LiveChat'. These can be added to the website respectively.

**Comment by psalmuelle** [comment]
We can do research on common questions asked by users and compile them into a page (FAQ). I want to work on this ticket @tuxology  with @comfy96 if you approve!

**Comment by comfy96** [comment]
That would be great! 
I'd be glad to work on it too @tuxology @psalmuelle

**Comment by psalmuelle** [comment]
Creating a live chat would be nice, but I think it would only work out if an admin is available to attend to users round the clock. A FAQ page should contain basic info users might want to know urgently. Any other enquiries can be directed to zubhub email.
What is your suggestion @tuxology @comfy96 ?

**Comment by comfy96** [comment]
> Creating a live chat would be nice, but I think it would only work out if an admin is available to attend to users round the clock. A FAQ page should contain basic info users might want to know urgently. Any other enquiries can be directed to zubhub email. What is your suggestion @tuxology @comfy96 ?

Livechat can use a bot.. instead of a real person @psalmuelle @tuxology

**Comment by the-techgurl** [comment]
@comfy96 great observation!

**Comment by comfy96** [comment]
> @comfy96 great observation!

Thank you @the-techgurl

**Comment by tuxology** [comment]
Please edit the description with a more concrete implementation/feature request along with mockups. Livechat like features have been requested earlier but we are not in favor of them right now given the privacy and moderation concerns. FAQs is something worth exploring if UX around it is good.

**Comment by comfy96** [comment]
> Please edit the description with a more concrete implementation/feature request along with mockups. Livechat like features have been requested earlier but we are not in favor of them right now given the privacy and moderation concerns. FAQs is something worth exploring if UX around it is good.

Noted, I'd come up with a good UX for the FAQs. Thank you  @tuxology @srish

**Comment by comfy96** [comment]
I noticed the Zubhub website doesn't have a frequently asked question, so I carry out UX research on this. I asked some kids from ages 5-10 years old to use the website on their own, as new users, they are supposed to signup. in the process of signing up, they kept asking me questions like:
1. what is an Email?
2. What is a username?
3. I don't have a phone number, what should I do?
4. What do they mean by 'location'?
5. What is a 'BIO'?
 I have to answer all of these questions for them. so I decided to come up with a FAQs page for the kids, parents, and educators, also added some questions that parents are likely to ask before giving their kids access to the website. This will make it easy for kids, and every other user to navigate through the website. The FAQ is going to be at the footer of the website as a link(when a user clicks on it, it takes them to the FAQs page).

Below is the wireframe 
![FAQs Wireframe 4 (2)](https://user-images.githubusercontent.com/57982384/226692567-49f126db-ded9-4b7c-bac1-8744e6950458.png)

Design:
![FAQs Design (1)](https://user-images.githubusercontent.com/57982384/226692681-e1985e0f-1358-43c1-8c58-26525b1b19b8.png)

FAQ added as a link: At the footer 

![Introduction 1](https://user-images.githubusercontent.com/57982384/228321103-bafb2f23-bde8-492d-ab96-4fa21000626a.png)



@tuxology @srish let me know what you think.
